### DISCUSSION TITLE
I want integrate the sandbox builder into my project

### DISCUSSION THREAD

**Discussion by imtag** [codesandbox/codesandbox-client#2224] [General]
# ❔ Question

Codesandbox is amazing!
I really want to integrate sandbox builder into my project, such as providing code, which can complete code  compile in browser, What should I do? Thank you.

**Comment by lbogdan** [comment]
Hey @sankincn ,

Does this https://github.com/codesandbox/codesandbox-client/blob/master/packages/sandpack/README.md look like something that would help?
